{"name": "# C5120 Logout from the Sidebar_menu and verify the login page will open after logout", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.TimeoutException: Message: \n\n", "trace": "  File \"c:\\users\\rapid\\appdata\\local\\programs\\python\\python38-32\\lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"c:\\users\\rapid\\appdata\\local\\programs\\python\\python38-32\\lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features\\steps\\C5120_Logout_from_the_Sidebar_menu.py\", line 16, in clck_lgt_btn\n    context.app.main_page.clck_lgt_btn()\n  File \"C:\\Everything\\IT\\Testing\\Automation_08_09_2019\\UIautomation_python\\pages\\main_page.py\", line 363, in clck_lgt_btn\n    wait.until(EC.element_to_be_clickable(LGT_BTN)).click()\n  File \"c:\\users\\rapid\\appdata\\local\\programs\\python\\python38-32\\lib\\site-packages\\selenium\\webdriver\\support\\wait.py\", line 80, in until\n    raise TimeoutException(message, screen, stacktrace)\n"}, "steps": [{"name": "Given Loginpage", "status": "passed", "start": 1604923625439, "stop": 1604923628278}, {"name": "Then Login with the given credentials", "status": "passed", "start": 1604923628279, "stop": 1604923631104}, {"name": "Then Click on the User name in the Sidebar menu", "status": "passed", "start": 1604923631104, "stop": 1604923633919}, {"name": "Then Hover over the Logout button in the dropdown menu and click on the button Logout", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.TimeoutException: Message: \n\n", "trace": "  File \"c:\\users\\rapid\\appdata\\local\\programs\\python\\python38-32\\lib\\site-packages\\behave\\model.py\", line 1329, in run\n    match.run(runner.context)\n  File \"c:\\users\\rapid\\appdata\\local\\programs\\python\\python38-32\\lib\\site-packages\\behave\\matchers.py\", line 98, in run\n    self.func(context, *args, **kwargs)\n  File \"features\\steps\\C5120_Logout_from_the_Sidebar_menu.py\", line 16, in clck_lgt_btn\n    context.app.main_page.clck_lgt_btn()\n  File \"C:\\Everything\\IT\\Testing\\Automation_08_09_2019\\UIautomation_python\\pages\\main_page.py\", line 363, in clck_lgt_btn\n    wait.until(EC.element_to_be_clickable(LGT_BTN)).click()\n  File \"c:\\users\\rapid\\appdata\\local\\programs\\python\\python38-32\\lib\\site-packages\\selenium\\webdriver\\support\\wait.py\", line 80, in until\n    raise TimeoutException(message, screen, stacktrace)\n"}, "start": 1604923633920, "stop": 1604923651256}, {"name": "And https://devcloud.connectedio.com/ is open", "status": "skipped", "start": 1604923653411, "stop": 1604923653411}], "start": 1604923620099, "stop": 1604923653411, "uuid": "0326bcc7-529d-4295-bdde-2ab88c187c0e", "historyId": "520fbb859be4ec07e48b2ac30200cc54", "labels": [{"name": "severity", "value": "normal"}, {"name": "feature", "value": "# C5120 Logout from the Sidebar_menu"}, {"name": "framework", "value": "behave"}, {"name": "language", "value": "cpython3"}]}